src/main/java/org/traccar/protocol/Gt06ProtocolDecoder.java

```diff
@@ -210,6 +210,11 @@ private void sendPhotoRequest(Channel channel, int pictureId) {
     }
 
     public static boolean decodeGps(Position position, ByteBuf buf, boolean hasLength, TimeZone timezone) {
+        return decodeGps(position, buf, hasLength, true, true, timezone);
+    }
+
+    public static boolean decodeGps(Position position, ByteBuf buf, boolean hasLength, boolean hasSatellites,
+                                    boolean hasSpeed, TimeZone timezone) {
```

Comment:
We don't do this formatting. Please move everything to the next line with 2x indentation. Check other code to see formatting style.